### PR TITLE
Fix `afvd` command for compound types

### LINKED ISSUE #1784: afvd Cannot find base type "char [20]"
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Fedora x64
| File format of the file you reverse (mandatory)      | ELF
| Architecture/bits of the file (mandatory)            | ELF/x86_64
| `rizin -v` full output, **not truncated** (mandatory)  | 0.3.0 @ linux-x86-64, package: 0.3.0-1.fc33 (Fedora)

### Expected behavior
No error and the variable displayed.

### Actual behavior
```
Cannot find base type "char [20]"
```

### Steps to reproduce the behavior
```
$ rizin -A ./test/bins/elf/ls
> s 0x00004070
> afvt var_18h "char [20]"
> afvd var_18h
```

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Some leftovers in the `afvd` command handlers.

**Test plan**

CI is green

**Closing issues**

Closes https://github.com/rizinorg/rizin/issues/1784

### CODE REVIEW

**Review by wargio — APPROVED** [review]

File: test/db/cmd/types [hunk 1]
```diff
@@ -1752,7 +1791,6 @@ EXPECT=<<EOF
 var char ** var_14e0h @ rbp-0x14e0
 var int var_14d4h @ rbp-0x14d4
 var int * wstatus @ rbp-0x14cc
-var int * options @ rbp-0x14c8
```
Comment (wargio) [inline test/db/cmd/types:1755]: why this change? is this a regression?

Comment (XVilka) [reply test/db/cmd/types:1755]: It's 64bit binary, pointer length detected as 8 bytes, thus this variable is overlapped with the previous `wstatus`.

File: librz/type/helpers.c [hunk 2]
```diff
@@ -340,6 +340,28 @@ RZ_API bool rz_type_is_identifier(RZ_NONNULL const RzType *type) {
 	return type->kind == RZ_TYPE_KIND_IDENTIFIER;
 }
 
+/**
+ * \brief Checks if the RzType is strictly atomic
```
Comment (ret2libc) [inline librz/type/helpers.c:344]: What's the difference between an "atomic" type and a "strictly atomic" one??

Comment (XVilka) [reply librz/type/helpers.c:344]: Atomic type (next function) is `char`, `char*`, `char****`, `char[20]`, `char[40][5][10]`, `char**[5][6]`, etc
Strictly atomic type is just `char`. Not a pointer or array of the atomic type.

**Pull request merged**